src/game_config.cpp

```diff
@@ -125,6 +125,13 @@ bool show_partial_orb;
 bool show_status_on_ally_orb;
 bool show_unmoved_orb;
 
+//
+// Reach Map UI flags
```

Comment:
Small nitpick, as these are not flags.
```suggestion
// Reach map opacity variables
```

Reply:
thanks, fixed